docs/getting-started/my-account/README.md

```diff
@@ -197,7 +186,7 @@ If you cannot see the entry **Delete account** in the **My account** side menu,
 
 To delete your account, select **Delete account** from the side menu and enter your login to confirm the deletion.
 
-![delete account](1572881984371.png)
+![delete-account](Bildschirmfoto 2022-03-02 um 16.15.19.png)
```

Comment:
please rename the image file and change the name here, so there are no spaces in it.